GiniVision/Classes/ReviewViewController.swift

```diff
@@ -16,13 +16,27 @@ import UIKit
 public typealias ReviewSuccessBlock = (_ imageData: Data) -> ()
 
 /**
+ Block which will be executed each time the user rotates a picture. It contains the JPEG representation of the image including meta information about the rotated image. In the case of a PDF, it should proceed to analysis screen once it has been validated.
```

Comment:
Block **that**

Also, why should clients validate the PDF? Shouldn't they only receive a valid one?

Reply:
As I explained before the code review, document validation should be also done on the ReviewViewController because the document could have been imported with _Open with..._.